.github/workflows/release.yml

```
@@ -15,7 +15,8 @@ jobs:
       - name: Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          fetch-depth: 0
+          fetch-depth: 1
+          persist-credentials: false
       - name: Set up Go
         uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # 5.1.0
```

This gets rid of the last `high` issue:
```suggestion
        uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # 5.1.0
        with:
          cache: false
```

---

added by [4b529ba](https://github.com/grafana/cloudcost-exporter/pull/483/commits/4b529ba289fe9879309b36ffc46c87b5e66bcce1)